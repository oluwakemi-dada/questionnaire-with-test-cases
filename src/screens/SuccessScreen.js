import React from 'react';
import styled from 'styled-components';
import { Wrapper } from '../components/form/FormResources';
import SuccessImg from '../assets/images/success.svg';

const Content = styled.div`
  margin-top: 5rem;

  p {
    text-align: center;
    font-size: 2.5rem;
  }
`;

const ImageContainer = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
  margin-top: 4rem;

  > * {
    width: 25%;

    @media (max-width: 1200px) {
      width: 30%;
    }

    @media (max-width: 768px) {
      width: 40%;
    }
  }
`;

const SuccessScreen = () => {
  return (
    <Wrapper>
      <Content>
        <p>Your submission was successful ðŸ˜‹</p>

        <ImageContainer>
          <img src={SuccessImg} alt='' />
        </ImageContainer>
      </Content>
    </Wrapper>
  );
};

export default SuccessScreen;
