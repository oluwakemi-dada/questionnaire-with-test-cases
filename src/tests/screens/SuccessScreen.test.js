import React from 'react';
import { shallow } from 'enzyme';
import SuccessScreen from '../../screens/SuccessScreen';

test('should render SuccessScreen correctly', () => {
  const wrapper = shallow(<SuccessScreen />);
  expect(wrapper).toMatchSnapshot();
});
