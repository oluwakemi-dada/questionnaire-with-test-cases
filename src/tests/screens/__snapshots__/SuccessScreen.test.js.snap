// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SuccessScreen correctly 1`] = `
<styled.div>
  <styled.div>
    <p>
      Your submission was successful ðŸ˜‹
    </p>
    <styled.div>
      <img
        alt=""
        src="success.svg"
      />
    </styled.div>
  </styled.div>
</styled.div>
`;
